### ISSUE TITLE
[1.18.1] Unable to attach vanilla lantern to bottom of post arm

### ISSUE THREAD

**Issue by ghost** [TwistedGate/ImmersivePosts#38]
### Description of the issue
I am able to attach a chain to the bottom of an arm, however nothing happens if I try to place a lantern.

### Versions and Modlist
Forge 39.0.88
Immersive Engineering 7.1.0-145
Immersive Posts 4.3.0-2
[Modlist](https://pastebin.com/dnM0WdKy)

**Comment by TwistedGate** [comment]
Yeah thats a little quirk that i can't fix.. (unless there is someone who knows a workaround, then i'll gladly accept a PR)

The vanilla lanterns require the bottom face of a block to have at least the size of a fence AND "touch" the ground.

The post arms only flip if there is a block already below that is a minimum of 6 pixels wide in both X and Z

So you'd have to place the Lantern first for it to flip, but since the lantern checks before placing if it can hang from the block above that will never ever happen.. it's an endless loop of "nope"

In short
![grafik](https://user-images.githubusercontent.com/31029668/169573814-38d005de-3340-45e4-bda9-9a362fce9134.png)
![grafik](https://user-images.githubusercontent.com/31029668/169574015-33d43062-43d1-4ace-86b4-f4e9c14a8201.png)

They both cancel each other basicly..
